packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -153,12 +153,15 @@
 
     &__menu-left {
         justify-content: flex-start;
-        align-items: center;
         order: -1;
         display: inline-flex;
         flex: 1;
         height: #{$HEADER_HEIGHT - 1px};
 
+        @include desktop {
+            align-items: center;
+        }
+
```

Comment:
Only `align-items: center` needs to be applied in desktop for this:
<img width="751" alt="image" src="https://github.com/binary-com/deriv-app/assets/103104395/2f5f62ed-918f-4a8a-81f6-714108aba4d4">